src/Voron/Data/BTrees/Tree.Stream.cs

```diff
@@ -257,6 +257,27 @@ public VoronStream ReadStream(Slice key)
             return new VoronStream(tree.Name, chunksDetails, _llt);
         }
 
+        /// <summary>
+        /// This method just verify that a stream exist under the given key
+        /// </summary>
+        /// <param name="key"> The key of the stream</param>
+        /// <returns>'true' if there is a stream under the given 'key' and 'false' otherwise</returns>
+        public bool SeekStream(Slice key)
```

Comment:
As the purpose of this method is to check the existence  of the stream we'll be better of by renaming it to `Exists` or something like that

Reply:
okay